src/zcl/definition/cluster.ts

```diff
@@ -4713,6 +4713,103 @@ const Cluster: {
         commands: {},
         commandsResponse: {},
     },
+    manuSpecificInovelliVZM31SN: {
+        ID: 64561,
+        manufacturerCode: 0x122f,
+        attributes: {
+            dimming_speed_up_remote: {ID: 0x001, type: DataType.uint8},
```

Comment:
Please make all attributes and commands camelcased, e.g. `dimming_speed_up_remote` -> `dimmingSpeedUpRemote`